src/ruby/yast/core_ext/string/ansi.rb

```diff
@@ -0,0 +1,18 @@
+class String
+  # Removes color and cursor escape codes (modifying the string)
+  # http://en.wikipedia.org/wiki/ANSI_escape_code
+  def remove_ansi_sequences
+    remove_ansi_color_sequences
+    remove_ansi_cursor_sequences
+  end
+
+  # Removes ANSI color escape codes (modifying the string)
+  def remove_ansi_color_sequences
```

Comment:
It does not hurt, IMO. I guess you always want to remove all ANSI sequences (color and cursor), but maybe sometimes you are only interested in one type. But I'm not strongly against offering only ```remove_ansi_sequences```